packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -205,6 +211,15 @@ export class CdkErrorMapper {
       errorName: 'GetLambdaLayerVersionError',
       classification: 'ERROR',
     },
+    {
+      errorRegex:
+        /User:(.*) is not authorized to perform:(.*) on resource:(.*) because no identity-based policy allows the (?<action>.*) action/,
+      humanReadableErrorMessage:
+        'Unable to deploy due to insufficient permissions',
+      resolutionMessage: 'Ensure you have permissions to call {action}',
```

Comment:
can we mention both action and resource ?

Reply:
Good catch it will be more helpful with resource included in the resolution, added in https://github.com/aws-amplify/amplify-backend/pull/2258/commits/549bc6e73899e43d46d1ea854460ab3729109a4f.